cmake/FindSCALAPACK.cmake

```diff
@@ -30,10 +30,14 @@ find_package_handle_standard_args(SCALAPACK DEFAULT_MSG SCALAPACK_LIBRARY)
 
 mark_as_advanced(SCALAPACK_LIBRARY)
 
+# TODO: Checks?
+mark_as_advanced(SCALAPACK_INCLUDE_DIR)
+
```

Comment:
Same question here as for `LAPACK_INCLUDE_DIR`.

Reply:
Not needed, as we don't include `Cblacs.h` or `mkl_blacs.h` directly.